lib/components/app/styled.js

```diff
@@ -0,0 +1,94 @@
+import styled, {css} from 'styled-components'
+
+import DateTimePreview from '../form/date-time-preview'
+
+const activeCss = css`
+  background: #e5e5e5;
+  -webkit-box-shadow: inset 0px 0px 5px #c1c1c1;
+     -moz-box-shadow: inset 0px 0px 5px #c1c1c1;
+          box-shadow: inset 0px 0px 5px #c1c1c1;
+   outline: none;
```

Comment:
Non blocking: some contrast or a different color would help distinguish pushed settings and date buttons from the default state.